shelterAPI.json

```diff
@@ -103,6 +100,38 @@
           }
         }
       }
+    },
+    "/shelter/announcements": {
+      "get": {
+        "tags": [
+          "Endpoints"
+        ],
+        "summary": "Gets all announcements for shelter. Requires shelter role. Gets shelter id from auth claims.",
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "404": {
+            "description": "Client error"
+          }
+        }
+      }
+    },
+    "/shelter/pets": {
```

Comment:
Same here

Reply:
Same as before, maybe it would be more intuitive to add a filtering option for GET /pet with an optional shelterId, rather than adding it in /shelter with a specific auth claim